src/modules/esl-panel-group/core/esl-panel-group.ts

```diff
@@ -212,14 +208,22 @@ export class ESLPanelGroup extends ESLBaseElement {
     } else {
       // set initial height
       this.style.height = `${from}px`;
-      // make sure that browser apply initial height to animate
+      // make sure that browser applies initial height to animate
       afterNextRender(() => {
         this.style.height = `${to}px`;
         this.fallbackAnimate();
       });
     }
   }
 
+  /** Check if transition happens and run afterAnimate step if no transition presented*/
```

Comment:
very minor cosmetic (if you suddenly change something else):
```suggestion
  /** Check if transition happens and run afterAnimate step if no transition presented */
```